src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
                                             MAV_CMD_DO_SET_CAM_TRIGG_DIST,
                                             MAV_FRAME_MISSION,
-                                            0.0,                    // trigger distance (off)
+                                            -1.0,                   // trigger distance (off)
```

Comment:
@LorenzMeier This is what turns the camera off for PX4. It doesn't break older PX4 either.